src/test/java/io/supertokens/test/WebserverTest.java

```diff
@@ -787,26 +787,6 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
 
     @Test
     public void invalidBasePathTest() throws InterruptedException, IOException {
-        {
-            Utils.setValueInConfig("base_path", "somepath/");
-            String[] args = { "../" };
-            TestingProcess process = TestingProcessManager.start(args);
-            EventAndException e = process.checkOrWaitForEvent(PROCESS_STATE.INIT_FAILURE);
-            assertTrue(e != null && e.exception instanceof QuitProgramException
-                    && e.exception.getMessage().equals("base_path must start with a '/'"));
-            Utils.reset();
-        }
-
-        {
-            Utils.setValueInConfig("base_path", "/somepath/");
-            String[] args = { "../" };
-            TestingProcess process = TestingProcessManager.start(args);
-            EventAndException e = process.checkOrWaitForEvent(PROCESS_STATE.INIT_FAILURE);
-            assertTrue(e != null && e.exception instanceof QuitProgramException
-                    && e.exception.getMessage().equals("base_path cannot end with '/'"));
-            Utils.reset();
-        }
-
```

Comment:
Since these tests are removed, we should add tests which sets the base path as `somepath/` and when calls `Config.getBasePath()`, it returns `"/somepath"`. Likewise for when the basepath is `/somepath/`

Reply:
I have reimplemented this functionality with calls to `getBasePath` instead of checking the exception.